docs/04_data_analysis_tools.md

```diff
@@ -485,3 +485,26 @@ On the model build screen, all the factors are displayed that are contained with
 The result view contains two tabs, one with the raw data, either BLUPS or BLUEs, and the other the adjusted means from the raw data.
 
 The results can be stored in the database as an analysis, by clicking the button provided on the top of the data.
+
+
+4.14 Genomic Prediction of Cross Performance (GCPC)
+--------------
+
+The GCPC tool is available at <a href="/tools/gcpc">/tools/gcpc</a> and a link is provided from the Analyze menu.
+The GCPC tool implements genomic prediction with additive and directional dominance in the linear mixed model to predict for cross performance.
+
+Before using the tool, first create a dataset using the Wizard containing the data that you would like to analyze. (The dataset should have genotyping_protocols).
+Second, create Selection Indices for your traits using Selection Index in Analyze Menu.
+
+To use the tool, Select the GCPC tool from the Analyze menu.
+
+Then, select the dataset with genotyping_protocols that you wish to analyze, click on "Proceed to Factor Selection" to load available factors that can be included in the model.
+
+Select the factors you wish to include in the model either as Fixed or Random. Click "None" for factors that you don't want to include in the model. Note that the "germplasmName" is factored as Random by default.
+
+The next step is to select the selection index for your traits on the dropdown menu.
+
+Once you are through, click "Run GCPC" to run the model. The output will be presented in form of a table with "ID", "Parent1", "Parent2" and their cross prediction merit organized in descending order.
+The results will also have sex information based on whether the dataset has plant sexes available in the database.
+
+<img src='{{"assets/images/gcpc.png" | relative_url }}' width="522" />
```

Comment:
@cmn92 I'm working on migrating the docs to a different theme, and realized the image referenced here must have been left out by accident. Can you go back and add it?